src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/refdata/VenueDataLoaderTest.java

```diff
@@ -29,6 +28,5 @@ public void shouldLoadReferenceData() {
         Map<String, VenueDetails> venueDetailsMap = venueDataLoader.getVenueDetailsMap();
 
         assertNotNull(venueDetailsMap);
-        assertEquals(265, venueDetailsMap.size());
```

Comment:
Why is this test removed? This is a valid test to make sure all the venues are there

Reply:
Yes, it needs to be changed to:
`assertEquals(266, venueDetailsMap.size());`
not deleted

Reply:
I asked for it to be removed. A test like this is pointless - it just means that whenever someone adds a new venue they now have to come and fix a test (in a completely different project, no less). If we only ever expected a set number of values I could maybe see the point in this but as it stands it has no use.

I'm even tempted to say we remove this test all together:
- It's not testing anything in case loader.
- The method is in sscs-common which has tests for it.
- It's only making an arbitrary size and not null assertion which doesn't give any real value.

Reply:
Would need to updated the test anytime a new venue is added, and this test would sneakily fail. so removing made sense to me as well